main.js

```diff
@@ -5,11 +5,52 @@ class Car {
     this.direction = direction
     this.location = location
   }
+
+  turn(direction) {
+    this.direction = direction
+    var img = document.querySelector('.car-img')
```

Comment:
You don't need to get the image from the DOM. It's already a property of `this`.